### PR TITLE
Fix: removes postgresql db uri MacOS specific syntax

### LINKED ISSUE #132: Bug: test_app_config.py contains bug for linux and windows OS contributors
**Describe the bug**
Contributors using linux and windows OS have different way to connect to the postgresql database compared to contributors on Mac OS.
On MacOS, `postgresql:///bit_schema` and `postgresql:///bit_schema_test` are used for this purpose
On Linux and Windows OS, the db_uri build using build_db_uri() method and would have the following syntax:
> {db_type_arg}://{db_user_arg}:{db_password_arg}@{db_endpoint_arg}/{db_name_arg}

However in the test_app_config.py, checking connection to postgresql are currently done using the syntaxes as per the MacOS only. 
Screen Shot 2020-09-11 at 9 06 42 am

Screen Shot 2020-09-11 at 9 06 51 am

This will cause the tests above to fail when contributors perform unittest by running 
> python -m unittest discover tests
.
**To Reproduce**
1. On linux or windows OS
2. run `python -m unittest discover tests`
3. The above tests will fail
* test_app_testing_config - on line 28
postgre error1

* test_app_development_config) - on line 102
postgre error2

**Expected behavior**
The tests should not be made for specific OS machine. So the 2 lines where specific machine syntax are used need to be removed or adjusted to work on any machine.

**Screenshots**
See screenshots above.

**Desktop (please complete the following information):**
 - OS: Windows OS
 - Browser NA (VS Code IDE)
 - Version ??

**Additional context**
NA

### PR DESCRIPTION
**Pull request by mtreacy002**
### Description
This PR removes MacOS specific syntax for connection to postgresql as explained in issue #132 

Fixes #132 

### Type of Change:
- Code
- Quality Assurance

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
Tested on Windows OS on Python test explorer VSCode IDE. All existing tests passed as a result.

### Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by mtreacy002** [comment]
@anitab-org/bridgeintech-maintainers or @anitab-org/coding-team. Can someone please give review on this? It's needed for tests run on VSCode IDE Windows OS to pass. Thanks.

**Review by rpattath — APPROVED** [review]

**Comment by mtreacy002** [comment]
Thank you @rpattath for your approval. @anitab-org/bridgeintech-maintainers, can someone please approve and merge this PR? I can't self-approve since I wrote this.

**Comment by mtreacy002** [comment]
@anitab-org/bridgeintech-maintainers  , I'm labelling issues with `Status: ready to merge` if the PR has already got approval by OS team but need an approval from other maintainer beside me. Please help approve and merge these PRs.

**Review by isabelcosta — APPROVED** [review]

**Pull request merged**
